app/controllers/api/v1/likes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Api::V1::LikesController < ApplicationController
+  before_action :find_photo, :authorized, :find_like
+
+  def destroy
+    if !Likes::AlreadyLikedService.new(like: @like, photo: @photo, user: @user).call || @like.nil?
+      render json: { message: 'Photo already unlike' }, status: :unprocessable_entity
+    else
+      Likes::DestroyerService.new(like: @like, photo: @photo, user: @user).destroy
```

Comment:
ServiceObject should contain only one `#call` method. As written below - we can use only one service to manage Likes